src/libraries/System.Numerics.Tensors/tests/TensorTests.cs

```diff
@@ -2701,7 +2735,7 @@ public void TensorObjectIndexerTests()
         [Fact]
         public void TensorGetPinnedHandleTests()
         {
-            Tensor<int> tensor = new Tensor<int>(new int[] { 1, 2, 3, 4 }, new nint[] { 2, 2 });
+            Tensor<int> tensor = new Tensor<int>(new int[] { 1, 2, 3, 4 }, new nint[] { 2, 2 }, 0);
```

Comment:
Do you have tests that pass anything else except 0? Negative numbers, positive numbers, int max, int min?

Reply:
Oh, right, sorry. Internal constructor. Nevermind.

Reply:
Added